### DISCUSSION TITLE
blog/2023/fosdem/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#66] [Blog comments]
# blog/2023/fosdem/

I recently attended FOSDEM 2023, my first FOSDEM! I had heard of the conference before, but hadn’t really looked into it too much. Fortunately, after some urging from friends and social media, I to...

https://chrisholdgraf.com/blog/2023/fosdem/

**Comment by thadguidry** [comment]
I reflected on your bullet points of "the same problems".  And after re-reading them it struck me that many (all?) of them could be addressed around this one:
* We systemically under-resource community guidance, management, and strategy efforts.

I have formed the opinion over many years that the "community" in open source becomes much more effective when there is a "leader" in the community.  I've heard arguments on both sides about having a leader at the top to begin with, but ultimately, it's the "style of leadership" that dispels a lot of the fears.
I think just like in companies, a *thoughtful, coaching leadership style can move mountains and spark motivation and happiness* at a corporate level, so to would that type of leadership style help many open source communities thrive again.
